### DISCUSSION TITLE
Issue with CI/CD

### DISCUSSION THREAD

**Discussion by cahillsf** [Homebrew/discussions#5197] [Everyday usage] [answered]
### Description of issue

It appears there's an issue with CI/CD in homebrew-core.  This PR to bump the clusterctl version (https://github.com/Homebrew/homebrew-core/pull/165897) has six queued jobs required to pass, but they are still waiting for runners.  Looking at the other open PRs, it appears this issue is affecting many PRs and seems to have started ~6 hrs ago.  heres are some other examples:
- https://github.com/Homebrew/homebrew-core/pull/165893
- https://github.com/Homebrew/homebrew-core/pull/165882

**Comment by ZhongRuoyu** [comment, marked as answer]
There was maintenance work on the CI infrastructure, so some CI jobs were suspended.

**Reply by cahillsf** [reply]
thanks -- was this communicated to the community?

**Reply by SMillerDev** [reply]
No, since for the community builds and reviews will be done when they are done. There is no guarantee of uptime for CI.

**Reply by cahillsf** [reply]
might be nice just for information sharing
